config/initializers/permissions_policy.rb

```diff
@@ -0,0 +1,11 @@
+# Define an application-wide HTTP permissions policy. For further
```

Comment:
Should we add this file if we're not going to use the functionality?

Reply:
I don't have a strong preference. I left it just as a reference but, as you said, we don't use it so I'll remove it to avoid any confusion.